splunk_handler/__init__.py

```diff
@@ -33,11 +33,14 @@ def __init__(self, host, port, username, password, index,
 
         # prevent infinite recursion by silencing requests and urllib3 loggers
         logging.getLogger('requests').propagate = False
-        logging.getLogger('urllib3').propagate = False
 
         # and do the same for ourselves
         logging.getLogger(__name__).propagate = False
 
+        # disable all warnings from urllib3 package
+        if not self.verify:
+            requests.packages.urllib3.disable_warnings()
```

Comment:
I'm afraid this is only going to silence the SSL verification warnings.  What about other warnings from the urllib3 library?  You'll possibly have an infinite recursion for things like connection failures, DNS lookup failures, etc.